### PR TITLE
add get_route function to network module

### PR DESCRIPTION
**Pull request by msciciel**
It would be nice to have a function which return routing information with help of 'ip route get <ip>' command.

Examples:

```
# salt-call network.get_route 127.0.0.1
local:
    ----------
    destination:
        127.0.0.1
    gateway:
        None
    interface:
        lo
    source:
        127.0.0.1

# salt-call network.get_route 10.10.10.10
local:
    ----------
    destination:
        10.10.10.10
    gateway:
        172.17.42.1
    interface:
        eth0
    source:
        172.17.0.115

# salt-call network.get_route 172.17.0.116
local:
    ----------
    destination:
        172.17.0.116
    gateway:
        None
    interface:
        eth0
    source:
        172.17.0.115

# salt-call network.get_route fe80::42:acff:fe11:73
local:
    ----------
    destination:
        fe80::42:acff:fe11:73
    gateway:
        None
    interface:
        lo
    source:
        fe80::42:acff:fe11:73
```

### CODE REVIEW

**Comment by jfindlay** [comment]
@msciciel, thanks for the feature.  In the future, please send features into the develop branch.

File: salt/modules/network.py [hunk 1]
```diff
@@ -1108,3 +1108,29 @@ def default_route(family=None):
                 ret.append(route)
 
     return ret
+
+
+def get_route(ip):
+    '''
+    Return routing information for given destination ip
+
+    CLI Example::
```
Comment (jfindlay) [inline salt/modules/network.py]: `.. versionadded:: 2015.5.3`

File: salt/modules/network.py [hunk 2]
```diff
@@ -1108,3 +1108,29 @@ def default_route(family=None):
                 ret.append(route)
 
     return ret
+
+
+def get_route(ip):
+    '''
+    Return routing information for given destination ip
+
```
Comment (jfindlay) [inline salt/modules/network.py]: `.. versionadded:: 2015.5.3`

**Comment by jfindlay** [comment]
@msciciel, there is a [lint error](https://jenkins.saltstack.com/job/salt-pr-lint-n/6668/violations/file/salt/modules/network.py/).

**Comment by msciciel** [comment]
I have removed empty lines at the end of file in my repo. Do I need make another pull request ?

**Comment by jfindlay** [comment]
@msciciel, no github will update the pull request automatically once you push your changes, thanks.

**Comment by msciciel** [comment]
Is ok now ? Because it's still error with:

jenkins/salt-pr-linode-cent7-n — Salt PR - Linode CentOS 7 #724 — FAILURE

**Comment by jfindlay** [comment]
@msciciel, those errors are unrelated.  Will you add a version added tag as described in https://github.com/saltstack/salt/pull/24747#discussion_r32640175?  Thanks.

**Comment by msciciel** [comment]
Check now if it's ok.

**Pull request merged**